src/deploy.ts

```diff
@@ -772,6 +776,38 @@ export async function promptDeployTarget(
     } else if (chosenProject !== null) {
       return {create: false, workspace, project: existingProjects.find((p) => p.slug === chosenProject)!};
     }
+  } else if (existingProjects.length >= 10) {
```

Comment:
Any harm in nesting `if (existingProjects.length < 10) {…} else {…}` here so that we don’t have to repeat the 10 twice and risk a chance that one of the conditions won’t be handled?